vuepress/docs/next/tutorials/devops/entando-de-app.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebarDepth: 2
+---
+
+# Configure the Entando App Context
+This tutorial describes how to modify the [EntandoApp CR](../../docs/consume/entandoapp-cr.md) to customize the ingress path of an Entando App Engine deployment. 
+
+## Prerequisites
+- An existing deployment of an Entando App or the ability to create one.
+    - If you haven't created a deployment or don't have a YAML file for an Entando deployment, follow the [Quickstart instructions](../../docs/getting-started/README.md).
+
+## Remove or Replace entando-de-app
+
+The ingress path of the App Engine currently defaults to **/entando-de-app**, which is visible in the URL. Follow the steps below to remove or replace this value.
+
+1. Open the `entando-app.yaml`, which contains several default parameters:
+
+```yaml
+apiVersion: entando.org/v1
+kind: EntandoApp
+metadata:
+  namespace: entando
+  name: quickstart
+spec:
+  environmentVariables: null
+  dbms: embedded
+  ingressHostName: YOUR-HOST-NAME
+  standardServerImage: eap
+  replicas: 1
+```
+
+2. Under `spec`, add the parameter `ingressPath`. If this parameter is not present, the path for the `entando-de-app` defaults to **/entando-de-app**.
```

Comment:
`entando-de-app` >> EntandoApp 

it sounded a bit funny repeated here

Reply:
i wanted to make the connection that the EntandoApp's server-side/consumer representation was the Entando App Engine, aka the entando-de-app... otherwise it's like, why this default? subtle and repetitive but technically correct and connects some dots...